related_website_sets.JSON

```diff
@@ -666,6 +666,26 @@
         "https://baomoi.com": "News Website owned by Zalo Group, VNG",
         "https://smoney.vn": "News Website owned by Zalo Group, VNG"
       }
+    },
+    {
+      "contact": "support@tolteck.com",
```

Comment:
Contrary to what I've[ previously said](https://github.com/tolteck/quetzal/pull/2404#discussion_r1565939602), a `contact` attribute is required...

Reply:
Sad...
How do you know? Is it `python check_sites.py` who complains?

Reply:
Yes, even though I was convinced that it was not required, the `check_sites.py` script complains about the missing contact attribute.

Reply:
My bad, it's not required for the `.well-known/related-website-sets.json` files served by the websites. In the current case, it's required.

Reply:
OK! Does it "complain" (like a warning) but returns a successful error code (== 0), or does it "error" with a non-zero error code?

Reply:
It looks like an exception is catched because of a failing [schema validation](https://github.com/tolteck/related-website-sets/blob/77eede393ec8a38dd6782d94931e74120cfc84d1/check_sites.py#L89). 

It seems that a text is printed and it stops the check (`return`), but doesn't exist with an error code different from 0. 

I've checked other PR on Google Chrome's repo and it seems that there's a Github Action that perform the check and if there's any output (other than success), it will fail.

As examples --> [^1]  [^2].

[^1]: https://github.com/GoogleChrome/related-website-sets/pull/346#issuecomment-2064203165
[^2]: https://github.com/GoogleChrome/related-website-sets/pull/346#issuecomment-2064203165